packages/message-manager/src/AbstractMessageManager.ts

```diff
@@ -37,6 +37,7 @@ export interface AbstractMessage {
   securityProviderResponse?: Record<string, Json>;
   metadata?: Json;
   error?: string;
+  messageParams?: any;
```

Comment:
Why do we need this on the `AbstractMessage`? Each concrete type adds their own correctly typed params, such as `PersonalMessage` defines the `messageParams` as `PersonalMessageParams`.